src/infrastructure/qrcode-reader.js

```diff
@@ -5,7 +5,7 @@ const readQR = imageBitmap =>
     const qr = new QrCode()
 
     qr.callback = (error, value) => {
-      if (error) return reject(error)
+      if (error) return reject(new Error(error))
```

Comment:
This isn't already an error that comes from qrcode-reader package?

Reply:
No, it's a String.

Reply:
I just discovered that, at least, 2 integration tests are passing without assertions checking. This is a problem and a bug.